scripts/apps/authoring/multiedit/multiedit.ts

```diff
@@ -189,7 +190,7 @@ function MultieditArticleDirective(authoring, content, multiEdit, lock, $timeout
             function openItem() {
                 authoring.open(scope.article).then((item) => {
                     scope.origItem = item;
-                    scope.item = _.create(item);
+                    scope.item = JSON.parse(JSON.stringify(item));
```

Comment:
wouldn't `{...item}` work here?

Reply:
It may or may not :) `{...item}` produces a shallow copy, so if nested objects got modified in the original, it would also be modified in the copy automatically, because they'd have the same reference
![screenshot-2020-09-15--14-42](https://user-images.githubusercontent.com/2076953/93211705-e428f800-f761-11ea-9629-d29c228412a5.png)

Reply:
yeah I know about that, I was just asking cause I thought `item` was not being modified at all but I guess it is

Reply:
The same object is used in article edit directive so I wouldn't be surprised if it was modified. I'm saving time and fixing it up front :smile: